specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/networkVirtualAppliance.json

```diff
@@ -716,6 +711,11 @@
     },
     "NetworkVirtualAppliancePropertiesFormat": {
       "properties": {
+        "nvaSku": {
```

Comment:
>"nvaSku" [](start = 8, length = 8)

this is moving because this scenario is unable to follow the standard sku contract?

Reply:
That is correct. The current ResourceSku model does not support scaleunits and vendor. These are unique identifiers for NVA sku customer is deploying. I explored looking at members like tier etc.. in the standard Sku contract, but using those fields means NVA Sku would diverge from VirtualHub scaleunits definition. We do not want to have divergent models for NVA with respect scale units definition that customers currently use for VPN and ER gateways.